modules/features/player/src/main/java/au/com/shiftyjelly/pocketcasts/player/view/PlayerHeaderFragment.kt

```diff
@@ -368,6 +369,7 @@ class PlayerHeaderFragment : BaseFragment(), PlayerClickListener {
 
             val imageBuilder: ImageRequest.Builder.() -> Unit = {
                 error(IR.drawable.defaultartwork_dark)
+                headers(headersOf("User-Agent", Settings.USER_AGENT_POCKETCASTS_SERVER))
```

Comment:
Would it be possible to include this header for the `Coil` client using `ImageLoader.Builder.callFactory()` and one of our `OkHttpClient` instances that already has an interceptor that attaches this header?

https://github.com/Automattic/pocket-casts-android/blob/bce43569b162d201706c7fd89fd8f85fd7c3a620/modules/services/ui/src/main/java/au/com/shiftyjelly/pocketcasts/ui/di/UiModule.kt#L17-L28

https://github.com/Automattic/pocket-casts-android/blob/bce43569b162d201706c7fd89fd8f85fd7c3a620/modules/services/servers/src/main/java/au/com/shiftyjelly/pocketcasts/servers/di/ServersModule.kt#L59-L63

Reply:
I like the idea. I might need to investigate if sharing one of the existing `OkHttpClient` instances with `Coil` can cause any regression (e.g. [with caching](https://rb.gy/hesrog)). I'll merge this PR for now and we can explore it separately.